datafusion/physical-plan/src/coalesce_partitions.rs

```diff
@@ -100,10 +100,6 @@ impl ExecutionPlan for CoalescePartitionsExec {
         None
     }
 
-    fn equivalence_properties(&self) -> EquivalenceProperties {
```

Comment:
It seems like the equivalence properties would not be changed by `CoalescePartitionsExec` (sort order would , of course)

Reply:
Thanks @alamb for catching, it. I implemented `CoalescePartitionsExec`, and it improved a test (😄 )